spec.bs

```diff
@@ -1196,8 +1199,14 @@ A <dfn export>fenced frame config instance</dfn> is a [=struct=] with the follow
 
   : <dfn>embedder shared storage context</dfn>
   :: null, or a [=string=]
-  
+
   : <dfn>is ad component</dfn>
+  :: A [=boolean=]
+
+  : <dfn>can disable untrusted network</dfn>
+  :: A [=boolean=]
```

Comment:
Do these ones not need defaults?

Reply:
The fields in `config instance` are always initialized by `instantiate a config`. I wanted to specify defaults for the fields in `config` so that if other specs don't fill them in when constructing their config they get default values. So I don't know if it's really "initial" values per se.

Reply:
OK. So should `has disabled untrusted network` not have `, initially false` below it then?

Reply:
That one is initially false because you always started with network disabled (has disabled=false), and then (if you can disable network) you later disable it (has disabled->true).

Reply:
changed to "initially", w/e